classes/controllers/grid/feature/selectableItems/SelectableItemsFeature.php

```diff
@@ -54,7 +54,8 @@ public function getInitializedRowInstance($args)
 
         if ($grid instanceof \PKP\controllers\grid\CategoryGridHandler) {
             $categoryId = $grid->getCurrentCategoryId();
-            $row->addFlag('selected', (bool) $grid->isDataElementInCategorySelected($categoryId, $row->getData()));
+            $data = $row->getData();
```

Comment:
`$row->getData()` returns a reference; is this change necessary?

Reply:
This is the signature of the `isDataElementInCategorySelected`:
```php
public function isDataElementInCategorySelected($categoryId, &$gridDataElement){}
```

So it requires a reference.

Reply:
Yes, and `GridRow::getData` returns a reference:

https://github.com/pkp/pkp-lib/blob/96b8b8e37bf197db6051d62700da2c63791f0de9/classes/controllers/grid/GridRow.php#L126

Reply:
Hmm, so it must be ok, I'll double check later 👌